src/workflows/foraging.bonsai.layout

```diff
@@ -45,7 +45,7 @@
       </Location>
       <Size>
         <Width>1436</Width>
-        <Height>660</Height>
+        <Height>659</Height>
```

Comment:
what in the layout is changing?

Reply:
This layout doesn't need to be changed. It's the layout used by me, not necessary for the behavior training. 

I will delete this PR and submit a new one to only change the codec.